sphinxlint/utils.py

```diff
@@ -7,6 +7,15 @@
 from sphinxlint import rst
 
 
+PER_FILE_CACHES = []
+
+
+def _per_file_cache(func):
```

Comment:
We have no API, remove the underscore?
```suggestion
def per_file_cache(func):
```

Reply:
Sure!